### PR TITLE
Update names-generator.go

### PR DESCRIPTION
**Pull request by sashakm**
added cédric villani and laika kudryavka to names list.

laika_ soviet_dog

### CODE REVIEW

**Comment by cpuguy83** [comment]
It seems a little off to include a dog here... Can you find a human woman to replace the dog?
Can you also be sure to include a wikipedia link?

File: pkg/namesgenerator/names-generator.go [hunk 1]
```diff
@@ -559,6 +562,9 @@ var (
 
 		// Christiane Nüsslein-Volhard - German biologist, won Nobel Prize in Physiology or Medicine in 1995 for research on the genetic control of embryonic development. https://en.wikipedia.org/wiki/Christiane_N%C3%BCsslein-Volhard
 		"volhard",
+		
+		// Cédric Villani - French mathematician, won Fields Medal, Fermat Prize and Poincaré Price for his work in differential geometry and statistical mechanics
```
Comment (thaJeztah) [inline pkg/namesgenerator/names-generator.go]: Can you add a link with more information as well (e.g. to wikipedia, as is used for other names?)

**Review by thaJeztah — CHANGES_REQUESTED** [review]

**Review by thaJeztah — CHANGES_REQUESTED** [review]
Thank you for updating! Can you:

- Squash your commits, so that there's only a single commit in this PR
- Update the commit message to match the changes?
- Update your sign-off to [include your real name, not your GitHub handle](https://github.com/moby/moby/blob/master/CONTRIBUTING.md#sign-your-work) 😅 

LGTM after that's done 👍

**Comment by thaJeztah** [comment]
oh, looks like you have to `gofmt` the file;

```
12:13:01 pkg/namesgenerator/names-generator.go:1::warning: file is not goimported (goimports)
12:13:01 pkg/namesgenerator/names-generator.go:1::warning: file is not gofmted with -s (gofmt)
12:13:02 Build step 'Execute shell' marked build as failure
```

**Comment by AkihiroSuda** [comment]
CI green

@thaJeztah LGTY?

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Review by yongtang — APPROVED** [review]
LGTM, thanks!

**Pull request merged**